File: py/regex/email.py
import sys
import re

def check_email(email):
  m = re.match('^([0-9a-z_]+[0-9a-z_+.]*)@([0-9a-z_]+[0-9a-z_+.]*\.[0-9a-z]+)$', email, re.I)
  if m:
    return (m.group(1), m.group(2)) # username and domain
  else:
    return None

if __name__ == '__main__':
  print(check_email(sys.argv[1]))

File: js/regex/email.js
'use strict';

function check_email(email) {
  return email.match(/^([0-9a-z_]+[0-9a-z_+.]*)@([0-9a-z_]+[0-9a-z_+.]*\.[0-9a-z]+)$/) !== null;
}

console.log(check_email(process.argv[2].toString()));